### ISSUE TITLE
Need an answer

### ISSUE THREAD

**Issue by Jakub41** [Glynk0613/ror-serversetting#1]
I was contacted by a guy from USA which tell me I'm responsible for this repo and I need to deleted or I will have problems with lawyers. 
Please back to me on my email
lemiszewski@gmx.com
Thank you

**Comment by tubbo** [comment]
![ballsy](https://user-images.githubusercontent.com/113026/51281508-98341780-19b0-11e9-8c55-f1880cc581e4.gif)

**Comment by tubbo** [comment]
@Jakub41 no, I'm not. I would imagine you know the owner of this repo better than I do because you worked on the same team.

**Comment by Jakub41** [comment]
I don't remember at all to ever worked with them where this is so I can check by myself?
